patches/common/chromium/fix_zoom_display.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Web Oct 31 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Web, 31 Oct 2018 09:08:02 -0700
+Subject: fix_zoom_display.patch
+
+This patch was released in 70.0.3512.0.
```

Comment:
Please link the Chromium patch that this backports!

Reply:
done!